docs/index.bs

```diff
@@ -2557,6 +2565,12 @@ spec: storage; urlPrefix: https://storage.spec.whatwg.org/
 
         Note: Keep this definition in sync with [=fetch a classic worker-imported script=].
 
+  A <dfn id="dfn-race-result">race result</dfn> is a tuple of a [=/race response=] and [=/used route=].
+
+  A [=/race result=] has an associated <dfn export id="dfn-race-response">race response</dfn> (a [=/response=]).
```

Comment:
Since "race response" has already been defined, and this is a race response within the race result, I suggest you to use `<dfn export for="race result">race response</dfn>` or so to avoid the name conflict.  Of course, you need to revise L2568 accordingly.